daffodil-cli/src/it/scala/org/apache/daffodil/executing/TestCLIexecuting.scala

```diff
@@ -1,245 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.daffodil.executing
-
-import org.junit.Assert._
-import org.junit.Test
-import org.apache.daffodil.CLI.Util
-import net.sf.expectit.matcher.Matchers.contains
-import net.sf.expectit.matcher.Matchers.matches
-import net.sf.expectit.ExpectIOException
-import org.apache.daffodil.Main.ExitCode
-
-class TestCLIexecuting {
-
-  val output3 = Util.getExpectedString("output3.txt")
-  val output13 = Util.getExpectedString("output13.txt", true)
-  val output14 = Util.getExpectedString("output14.txt", true)
-  val output15 = Util.getExpectedString("output15.txt", true)
-  val output16 = Util.getExpectedString("output16.txt", true)
```

Comment:
The `TestCLIexecuting` and `TestCLIlisting` files were both really about running the CLI "test" subcommand to run TDML tests, with one focused on executing tdml tests and the other focused on listing tdml tests. And in fact, the "executing" file even had multiple tests that were just listing tests, and I found some tests that were duplicated in the two files, so the intended distinction wasn't even clear. I also think "executing" also very clear since that could be about executing anything, like  parse, unparse, code generator, etc.

So to make it more obvious that this is the file for the "test" command, I combined these two files into a "TestCLItdml" file (I thought "TestCLItest" might be confusing?) and removed duplicates when I found them.